src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/index.tsx

```diff
@@ -155,7 +155,7 @@ const SampleIdInput = ({
         variant="outlined"
         rows={!isInEditMode ? 4 : 3}
         value={isInEditMode ? inputValue : inputDisplayValue}
-        placeholder="e.g. U12345, AF123456"
```

Comment:
this is the examples in the tree build force include sample box. It's shared between SARS-CoV-2 build and mpox. SARS-CoV-2 need `USA/CA-CZB-0000/2021` and mpox need `AF123456`